tests/unit/swagger/definitionsGeneration/metadata.spec.ts

```diff
@@ -474,6 +476,27 @@ describe('Metadata generation', () => {
       expect(parameter.parameterName).to.equal('firstname');
       expect(parameter.required).to.be.true;
     });
+
+    it('Should inline enums for TS Enums in path, query and header when using Swagger', () => {
+      const spec = new SpecGenerator2(parameterMetadata, getDefaultOptions()).GetSpec();
+      const method = spec.paths['/ParameterTest/Path/{firstname}/{last_name}/{age}/{weight}/{human}/{gender}']['get'];
```

Comment:
I'm a bit confused by the use of curly braces. Is this string interpolation and you forgot to include a dollar sign? Like {x} instead of ${x}? Sorry for not understanding what's happening here.

Reply:
It's the wat to get to the part of the spec for [this](https://github.com/lukeautry/tsoa/blob/master/tests/fixtures/controllers/parameterController.ts#L54) controller method

Reply:
Oh wow, yea. I really am "out of it." Hopefully I can convince my day job to five me a TypeScript project again soon.